packages/i18n/data/core.json

```diff
@@ -40,6 +40,7 @@
   "LoginForm.forgotPassword": "Forgot password?",
   "LoginForm.forgotPasswordTitle": "We are redirecting you to the Forgot Password form in {countdown} seconds.",
   "LoginForm.InvalidCredentials": "Sorry, but the login details entered below are not in our system. Please enter your correct email address and password below.",
+  "LoginForm.invalidEmail": "Please enter a valid Email address.",
```

Comment:
one day we will have translations that don't have German capitalization rules in English 😆